trusty wants an extra header included, newer distributions work without.

--- a/sqlite.cc
+++ b/sqlite.cc
@@ -1,5 +1,6 @@
 #include <stdexcept>
 #include <cassert>
+#include <cstring>
 #include "sqlite.hh"
 #include <iostream>
 
